src/sidebar.c

```diff
@@ -1649,8 +1649,7 @@ void sidebar_focus_symbols_tab(void)
 	if (ui_prefs.sidebar_visible && interface_prefs.sidebar_symbol_visible)
 	{
 		GtkNotebook *notebook = GTK_NOTEBOOK(main_widgets.sidebar_notebook);
-		GtkWidget *symbol_list_scrollwin = gtk_notebook_get_nth_page(notebook, TREEVIEW_SYMBOL);
-
+		GtkWidget *symbol_list_scrollwin = ui_lookup_widget(main_widgets.window, "scrolledwindow2");
```

Comment:
Please keep the blank line between the variable declaration block and the rest (yes, it's kind of the same here, but that's currently how we do things everywhere else)

```suggestion
		GtkWidget *symbol_list_scrollwin = ui_lookup_widget(main_widgets.window, "scrolledwindow2");

```